### ISSUE TITLE
NaN Loss

### ISSUE THREAD

**Issue by gracecocks** [qfgaohao/pytorch-ssd#139]
I am training this for a custom dataset, but it seems that the loss is always nan

The command being used is: 

> python train_ssd.py --dataset_type pedestrian --datasets /content/pytorch-ssd/vision/datasets/Pedestrian/ --validation_dataset /content/pytorch-ssd/vision/datasets/Pedestrian/ --net mb2-ssd-lite --base_net models/mb2-imagenet-71_8.pth --batch_size 16 --num_epochs 100 --scheduler cosine --lr 0.0001 --t_max 200

I already followed a similar issue and decreased the learning rate but it still seems to have this issue. Is there a fix for this?
![nan](https://user-images.githubusercontent.com/48301204/96574707-4ad69e00-132c-11eb-9a8f-511db5e52fab.PNG)

**Comment by Alex-Beh** [comment]
Hello, have you solved your issue?

**Comment by marbali8** [comment]
Did you solve it?

**Comment by lakshayc-ss** [comment]
I also faced the same issue, This happened due to wrong labels.
So skip the files where xmin>xmax and ymin>ymax

Go to open_images.py and add this on line 77
```

            flag = False
            for i in boxes:
                if i[0]>i[2] or i[1]>i[3]:
                    flag = True
                    break
            
            if flag:
                continue

```
